### PR TITLE
chore: clarify the JMH Readme

### PR DESCRIPTION
**Pull request by Roiocam**
The original Readme line break seemed like was broken, it was not friendly for newcomers, I have clarified it both with shell and one-line command execution.

### CODE REVIEW

File: bench-jmh/README.md [hunk 1]
```diff
@@ -6,8 +6,18 @@ own jmh module)
 
 You can run them like:
 
-   project bench-jmh
-   jmh:run -i 3 -wi 3 -f 1 .*ActorCreationBenchmark
+```shell
+sbt shell
+pekko > project bench-jmh
+sbt:pekko-bench-jmh> jmh:run -i 3 -wi 3 -f 1 .*ActorCreationBenchmark
```
Comment (He-Pin) [inline bench-jmh/README.md:12]: add a link to `sbt-jmh`?
and it's `bench-jmh` not `pekko-bench-jmh` now.

and change `jmh:run` to `Jmh/run`

Comment (Roiocam) [reply bench-jmh/README.md]: Done.  

`pekko-bench-jmh` is just the real-world execution result, I guess it was the name of the sub-module `bench-jmh`

File: bench-jmh/README.md [hunk 2]
```diff
@@ -4,12 +4,22 @@ This subproject contains some microbenchmarks excercising key parts of Apache Pe
...
 
-You can run them like:
+Pekko uses [sbt-jmh](https://github.com/sbt/sbt-jmh) to integrate [Java Microbenchmark Harness](https://github.com/openjdk/jmh). You can run them like:
 
-   project bench-jmh
-   jmh:run -i 3 -wi 3 -f 1 .*ActorCreationBenchmark
+```shell
+sbt shell
+pekko > project bench-jmh
+sbt:bench-jmh> Jmh:run -i 3 -wi 3 -f 1 .*ActorCreationBenchmark
```
[the remark this answers is not in the document]
Comment (Roiocam) [reply bench-jmh/README.md:12]: Thanks, forgot to change that.

**Review by He-Pin — APPROVED** [review]
lgtm

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**